ovs/extensions/healthcheck/openvstorage/openvstoragecluster_health_check.py

```diff
@@ -513,18 +513,15 @@ def check_model_consistency(logger):
                 missing_in_model = []
 
                 # fetch configfile of vpool for the volumedriver
-                config_file = ConfigurationManager.get_config_file_path(product=ConfigurationProduct.VPOOL,
-                                                                        vpool_guid=vp.guid,
-                                                                        vpool_name=vp.name,
-                                                                        node_id=OpenvStorageHealthCheck.MACHINE_ID)
+                config_file = Configuration.get_configuration_path('/ovs/vpools/{0}/hosts/{1}{2}/config'.format(vp.guid, vp.name, OpenvStorageHealthCheck.LOCAL_ID))
```

Comment:
You should check whether any storagedriver is linked to the vpool and use vp.storagedrivers[0].name here instead of vp.name, OpenvStorageHealthCheck.LOCAL_ID
If the local node on which you are executing this HealthCheck is not running this vPool, the healthcheck will reports unnecessary failures